docker-build/build.sh

```diff
@@ -0,0 +1 @@
+docker build -t invoke_ai -f Dockerfile ..
```

Comment:
how does this ensure to work when it is run from the Repository Root?

Reply:
This was added because the dockerfile needed to be run from root, as I don't want to git pull everything. This specifically sets the tag for the image and is expected to be run from within the docker_build folder itself. I might have to try and update it so it can be run from either location.